docs/reference/tools-reference/python-tool.md

```diff
@@ -12,6 +12,25 @@ Users can effortlessly create Python tools, edit code, and verify results with e
 | Inputs | -      | List of tool function parameters and its assignments | -       |
 
 
+### Types
+
+| Type | Python annotation             | Description                                |
+| ---- |-------------------------------|--------------------------------------------|
+| int | param: int                    | Integer type                               |
+| bool | param: bool                   | Boolean type                               |
+| string | param: str                    | String type                                |
+| double | param: float                  | Double type                                |
+| list | param: list or param: List[T] | List type                                  |
+| object | param: dict or param: Dict[K, V] | Object type                                |
+| xxConnection | param: xxConnection | Connection type, will be handled specially |
+
+
+Parameters with `Connection` type annotation will be treated as connection inputs, promptflow will try to find
```

Comment:
What does this sentence mean? I think UX filters by connection type and fills in connection name.

Reply:
It explains how does promptflow executor do -- try to find the connection with the input value as name, this line is added because there is a customer asked something like -- "why a magic string in flow yaml can be a connection?" We are trying to explain the magic string, I'll move the 'during execution time' forward to emphasize it.